docs/user-guide/connectors/slack.rst

```diff
@@ -31,9 +31,6 @@ Getting Credentials
      with ``xoxb``.
 
 
-For more detailed steps, visit the
-`Slack API docs <https://api.slack.com/incoming-webhooks>`_.
-
```

Comment:
More confusing than helpful. We don't actually use incoming webhooks for our bots. But i couldn't find a better slack resource on adding bot apps.

Reply:
hmm but we need some sort of docs to link to here no?

Reply:
found a better one: https://api.slack.com/events-api because we use the events API, it details about the "challenge", etc.